compiler/prelude/prelude.go

```diff
@@ -23,3 +24,21 @@ var jsmapping string
 
 //go:embed goroutines.js
 var goroutines string
+
+func Minified() string {
+	result := api.Transform(Prelude, api.TransformOptions{
+		Target:            api.ES2015,
+		MinifyWhitespace:  true,
+		MinifyIdentifiers: true,
+		MinifySyntax:      true,
+		KeepNames:         true,
+		Charset:           api.CharsetUTF8,
+		LegalComments:     api.LegalCommentsEndOfFile,
+	})
+	if len(result.Errors) > 0 {
+		e := result.Errors[0]
+		panic(fmt.Sprintf("%d:%d: %s\n%s\n", e.Location.Line, e.Location.Column, e.Text, e.Location.LineText))
+	}
```

Comment:
Maybe `glog.Errorf()` all the errors and then `glog.Fatalf()` with a general error to abort the program?

Reply:
I made a similar change now. I also log warnings, although that may not be ideal, as it will appear as noise to users of gopherjs, but only developers will be able to fix it. WDYT?

Reply:
I think we definitely want to log as many details as possible. The only way the users would see this error is if we ship a badly broken release, in which case getting as much debugging information to us with as few roundtrips as possible would help fix the issue faster.